### ISSUE TITLE
Botanical garden collections with the location in the garden

### ISSUE THREAD

**Issue by MattBlissett** [gbif/portal-feedback#1627]
The Darwin Core standard doesn't specify it, but our recommendation when publishing data about a botanical garden collection (or zoo, butterfly garden etc) is to give the co-ordinates of where the plant was collected.

Giving the exact location within the garden is confusing when other occurrences are present in the same analysis, as it appears that there are a lot of non-native species.

https://www.gbif.org/dataset/b7ef1d60-b0a0-11dd-aa14-b8a03c50a862
https://www.gbif.org/dataset/c5da77b7-7423-49a4-a379-f423ea0f07b8
https://www.gbif.org/dataset/1cb0f057-172b-4c10-99d8-cd38966dab31
https://www.gbif.org/dataset/ea05dcbd-0d05-4785-9313-0ec68cb90426

These were easily spotted with the classic hexagon map view restricted to living specimens on the homepage. The red hexagons stand out.

Another way, which would find very small datasets, would be looking at capabilities.json for min/max values to be the same.

https://api.gbif.org/v2/map/occurrence/density/capabilities.json?datasetKey=c5da77b7-7423-49a4-a379-f423ea0f07b8

**Comment by ManonGros** [comment]
Isn't it why they should choose "livingSpecimen" as basis of record?

Can we ask the publishers to put the location of original specimen was collected? What about the dates? What if some seeds were collected 100 ago somewhere but it is not longer occurring there? 

Wouldn't we have the same problem with a "zoo" dataset?

**Comment by MattBlissett** [comment]
If a botanist takes a few leaves from a plant to make into a herbarium voucher, some cuttings to put in a tissue collection, some seeds and the living plant, they have at least four occurrences. (A zoologist can do something similar with feathers/hair/droppings, a DNA sample, and the animal.)

- The herbarium voucher is a preserved specimen (often more than one, as leaves are sent to other herbaria)
- The tissue could reasonably be a preserved specimen or a living specimen; this is left to the judgement of the collection manager.
- The seeds could also be a living specimen or a preserved specimen
- The plant is a living specimen

All of them will have the same recordedBy, recordNumber, eventDate, location, coordinates and organismId, but different occurrenceIds / institution-collection-catalog numbers.

The plant may reproduce, seeds may be taken and stored in a living collection, and dead samples (seeds, leaves) might be stored. Eventually it will die (or be stolen!).

DWC doesn't have a term to represent this, but in ABCD the term `BotanicalGardenUnit/ProvenanceCategory` would be mapped to:

* W (Accession of wild source)
* Z (Propagule/s from a wild source plant in cultivation)
* G (Accession not of wild source)
* U (Insufficient data to determine which category applies)

depending on whether it's the original plant or its progeny.

[Royal Botanical Gardens Edinburgh](https://www.gbif.org/dataset/7adf20e0-c955-11de-95c0-b8a03c50a862) have mapped it to `samplingProtocol`, so there are values like "collected directly from the wild" or "from a cultivated plant of known (indirect) wild origin". They've also mapped `disposition` to either "alive" or "dead". These mappings seem like reasonable choices from the available DWC terms.

I'm sure other people have thought about this more than me, but I can't find anything relevant online.

**Comment by mdoering** [comment]
[dwc:establishmentMeans](http://rs.tdwg.org/dwc/terms/#dwc:establishmentMeans) to the rescue. Basis of record is not meant to distinguish native occurrences.

DwC clearly conflates event date and location as it can be the location & date you have seen the specimen or when you have seen it in the "wild" when collecting. You could argue to take a picture of a herbarium sheet and put the museum collection location on it. Its exactly the same for living collections. There are always at least 2 events, I guess even dwc:occurrences of the same organism. 

With propagated plants, germinated seeds and animals raised in zoos it does get trickier cause they never occurred in the wild. 

GBIF badly needs to come up with a very clear policy how these situation should be handled in the various standards, but especially in DwC. And how you can search for occurrences excluding non native ones.

**Comment by MattBlissett** [comment]
I don't think establishmentMeans solves it. Whether something is dead in the herbarium or outside in the garden, it can have been invasive when it was collected etc.

The exact location in the garden is about as useful as the cupboard in the herbarium.

https://github.com/gbif/ipt/wiki/IPT2ManualManageResources.wiki has advice on disposition and preparation, but not for living specimens.

**Comment by mdoering** [comment]
There needs to be an indication to know if the occurrence is cultivated/managed/captive. Thats the kind of records people like to filter out. Whether invasive is not relevant. And it will be very hard to judge in many cases whether something is native, depending a lot on the definition on native. 

But if a living organism is currently and actively supported by humans to survive at its location then this should be indicated *somehow*. It is a grey area.

iNaturalist offers a single flag to indicate this and I believe GBIF should have the same: 
https://www.inaturalist.org/pages/help#captive

**Comment by ManonGros** [comment]
@MattBlissett I don't think we could reasonably ask the publishers to provide then coordinates of where the specimen was originally collected. I think having the basis of record together with the appropriate establishmentMeans would be enough.
Should I close this issue?

**Comment by MattBlissett** [comment]
> GBIF badly needs to come up with a very clear policy how these situation should be handled in the various standards, but especially in DwC.

I think this is what's needed, so let's leave this open until that's done.

**Comment by mdoering** [comment]
Maybe we should propose a new dwc term with a simple boolean for captive / cultivated like  https://www.inaturalist.org/pages/help#captive ? All these multi value terms are blurring the picture.

**Comment by mdoering** [comment]
Alternatively GBIF could try to interpret such a flag based on various rules combining different occurrence properties, dataset or even record annotations by users . We could interpret a combination of establishmentMeans & basisOfRecord. Or use even have shapes for known botanical or zoological gardens and combine them with kingdom Animal & Plants, but exclude Arthropods and some other groups with small, mobile organisms. Having a little button in the portal to flag records would be useful. I can see this getting pretty well curated through different means and ultimately offer the best way to differ between wild and captive organisms.
